infra/embedding_step_functions/unified_embedding/handlers/compaction.py

```
@@ -2,43 +2,41 @@
-from logging import INFO, Formatter, StreamHandler, getLogger
 from typing import Any, Dict, List
+import utils.logging
 
 import boto3
 import chromadb
 
 # Import receipt_dynamo for proper DynamoDB operations
 from receipt_dynamo.data.dynamo_client import DynamoClient
+from receipt_dynamo.constants import ChromaDBCollection
 from receipt_label.utils.lock_manager import LockManager
+from receipt_label.vector_store import VectorClient
```

Ruff (0.12.2)

`receipt_label.vector_store.VectorClient` imported but unused

Remove unused import: `receipt_label.vector_store.VectorClient`

(F401)